app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -1,22 +1,22 @@
-<p>Welcome, <%= @resource.name %>!</p>
-
 <% account_user = @resource&.account_users&.first %>
```

Comment:
@tejaswinichile Can you help me verify if the refactor looks good here? Have I missed any condition here